packages/ui/src/components/filters/FilterContainer.tsx

```diff
@@ -0,0 +1,102 @@
+import React, {useState} from 'react';
+import {Collapse} from 'antd';
+import {SearchOutlined} from '@ant-design/icons';
+
+import {IFilter, IFilterGroup, onChangeType, VisualType} from './types';
+import StackLayout from "../../layout/StackLayout";
+import FilterSelector from "./FilterSelector";
+
+import styles from '@ferlab/style/components/filters/FilterContainer.module.scss';
+
+const { Panel } = Collapse;
+
+type FilterContainerProps = {
+    filterGroup: IFilterGroup;
```

Comment:
why did we remove the dictionary type ? I think it was useful for translations

Reply:
now its impossible to translate

Reply:
in KF we do not have translations. we dont need a dictionary. I think the translation should be done in cqdg in the input should be the result of that.